content/en/docs/reference/commands/operator/index.html

```diff
@@ -874,7 +874,7 @@ <h2 id="envvars">Environment variables</h2>
 <td><code>UNSAFE_ENABLE_ADMIN_ENDPOINTS</code></td>
 <td>Boolean</td>
 <td><code>false</code></td>
-<td>If this is set to true, dangerous admin endpoints will be exposed on the debug interface. Not recommended for production.</td>
+<td>If this is set to true, dangerous admin endpoins will be exposed on the debug interface. Not recommended for production.</td>
```

Comment:
Are these three typos to be expected?
Do they need to be manually fixed on a new branch if so?

Reply:
I made a comment to cherry-pick the fix into 1.12.